### PR TITLE
Add VxWorks support

### PR DESCRIPTION
**Pull request by wenshan1**
This PR adds VxWorks support.

### CODE REVIEW

**Review by derekmauro — CHANGES_REQUESTED** [review]
Changes to code code under `absl/time/internal/cctz/` must be sent to https://github.com/google/cctz.

File: absl/base/config.h [hunk 1]
```diff
@@ -397,6 +397,7 @@ static_assert(ABSL_INTERNAL_INLINE_NAMESPACE_STR[0] != 'h' ||
 //   AsmJS                             __asmjs__
 //   WebAssembly                       __wasm__
 //   Fuchsia                           __Fuchsia__
+//   VxWorks                           __VXWORKS__
```
Comment (derekmauro) [inline absl/base/config.h]: Please remove this. This is not a supported platform. Support for this platform is best-effort.

Comment (wenshan1) [reply absl/base/config.h]: Thank you for your quick review.  I have changed source code to solve it.

**Review by wenshan1 — COMMENTED** [review]
Changes to code under absl/time/internal/cctz/ was removed.

**Pull request merged**